packages/core/admin/admin/src/components/MainNav/NavUser.tsx

```diff
@@ -1,31 +1,52 @@
 import * as React from 'react';
-
-import { Flex, Menu, ButtonProps, VisuallyHidden, Avatar } from '@strapi/design-system';
-import { SignOut } from '@strapi/icons';
+import {
+  Flex,
+  Menu,
+  ButtonProps,
+  VisuallyHidden,
+  Avatar,
+  Typography,
+  Badge,
+} from '@strapi/design-system';
 import { useIntl } from 'react-intl';
 import { useNavigate } from 'react-router-dom';
 import { styled } from 'styled-components';
 
 import { useAuth } from '../../features/Auth';
 
-export interface NavUserProps extends ButtonProps {
-  initials: string;
-  children?: React.ReactNode;
-}
-
 const MenuTrigger = styled(Menu.Trigger)`
   height: ${({ theme }) => theme.spaces[7]};
   width: ${({ theme }) => theme.spaces[7]};
   border: none;
   border-radius: 50%;
-  // Removes inherited 16px padding
   padding: 0;
-  // Prevent empty pixel from appearing below the main nav
   overflow: hidden;
 `;
 
 const MenuContent = styled(Menu.Content)`
-  left: ${({ theme }) => theme.spaces[3]};
+  padding: 0;
+  max-height: 300px;
+  // min-width: 240px;
+  width: max-content;
+  max-width: 240px;
+`;
+
+const UserInfo = styled(Flex)`
+  && {
+    padding: ${({ theme }) => theme.spaces[4]};
```

Comment:
```suggestion
    padding: ${({ theme }) => theme.spaces[3]};
```

Since there's already a padding of `1` on `Menu.Content`, we can make this padding a bit smaller.